it-and-security/lib/macos-device-health.policies.yml

```diff
@@ -17,19 +17,7 @@
   resolution: An an IT admin, deploy a macOS, login window profile with the DisableGuestAccount option set to true.
   platform: darwin
 - name: macOS - Require 10 character password
-  query: SELECT 1 WHERE 
-    EXISTS (
-      SELECT 1 FROM managed_policies WHERE 
-        domain='com.apple.screensaver' AND 
-        name='askForPassword' AND 
-        CAST(value AS INT)
-    ) 
-    AND EXISTS (
-      SELECT 1 FROM managed_policies WHERE 
-        domain='com.apple.screensaver' AND 
-        name='minLength' AND 
-        CAST(value AS INT) <= 10
-    );
+  query: SELECT 1 FROM plist WHERE path='/Library/Preferences/com.apple.loginwindow.plist' AND key='GuestEnabled' AND value = 0;
```

Comment:
Hey @sharon-fdm did you mean to make this change?

Looks like we're adding the query to check for the guest account.

Reply:
Crap.
That's a mistake (that was also merged into main.)
I'll fix both and send a PR for Main as well.